### PR TITLE
Feat/drug addiction

### PR DESCRIPTION
**Pull request by leneherng**
### Changes
1. Drug Addiction dashboard
2. DB index add `fillColor` for AgencyIcon

### CODE REVIEW

File: apps/app/dashboards/public-safety/drug-addiction/index.tsx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+                            data: coordinate[key],
+                            label: t(key),
+                            borderColor: AKSARA_COLOR.DANGER,
+                            backgroundColor: AKSARA_COLOR.DANGER_H,
+                            fill: true,
+                            borderWidth: 1.5,
+                          },
+                        ],
+                      }}
+                    />
```
Comment (irfancoder) [inline apps/app/dashboards/public-safety/drug-addiction/index.tsx:100]: add this: `enableAnimation={!play}`

File: apps/app/dashboards/index.tsx [hunk 2]
```diff
@@ -73,6 +75,8 @@ const DashboardIndex: FunctionComponent<DashboardIndexProps> = ({
     for (const [category, dbs] of Object.entries(dashboards)) {
       const dashboards = dbs
         .filter(d => {
+          if (d.name === "covid-vaccination") Object.assign(d, { fillColor: AKSARA_COLOR.GREEN });
+          else if (d.agency === "aadk") Object.assign(d, { fillColor: AKSARA_COLOR.DANGER });
```
Comment (irfancoder) [inline apps/app/dashboards/index.tsx:79]: Can get messy really quick if you have more conditional coloring like this. Something to think about later

Comment (leneherng) [reply apps/app/dashboards/index.tsx:79]: the other method would probably to add a prop on data side?

Comment (irfancoder) [reply apps/app/dashboards/index.tsx:79]: Not really (reason: keep styling config on FE). Take a step back and think it through. For now, i'll accept it, but this is for you to explore in your free time.

**Review by irfancoder — APPROVED** [review]

**Pull request merged**